cli/aws_ddk/data/project_templates/ddk_lib/{{cookiecutter.directory_name}}/app.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+import aws_cdk as cdk
+from aws_cdk.aws_codebuild import BuildEnvironment, BuildEnvironmentVariable
+from aws_cdk.aws_iam import Effect, PolicyStatement
+from aws_cdk.pipelines import CodeBuildStep, ShellStep
+from aws_ddk_core.cicd import CICDPipelineStack
+from aws_ddk_core.config import Config
+from ddk_app.artifactory_stack import ArtifactoryStack
+
+app = cdk.App()
+
+# Artifactory parameters
+ENVIRONMENT_ID = "dev"
+DOMAIN_NAME = "ddk-lib-domain"
+DOMAIN_OWNER = Config().get_env(environment_id=ENVIRONMENT_ID).account
+REPOSITORY_NAME = "ddk-lib-repository"
+PIPELINE_NAME = "ddk-lib-pipeline"
+
+# Private artifactory stack
+artifactory_stack: ArtifactoryStack = ArtifactoryStack(
```

Comment:
This is only an example to get the discussion going. Both artifactory and CI/CD pipeline publish stage would be moved to core library

Reply:
Honestly I'm even considering to just keep this in the template, perhaps until there is an L2-L3 construct
Anyway open to suggestions